File: backend/src/main/java/com/ssafy/smartstore/dto/Store.java
package com.ssafy.smartstore.dto;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "store")
public class Store {
    @Id
    private Integer id;
    private String name;
    private Double latitude;
    private Double longitude;
    
	public Integer getId() {
		return id;
	}
	public void setId(Integer id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public Double getLatitude() {
		return latitude;
	}
	public void setLatitude(Double latitude) {
		this.latitude = latitude;
	}
	public Double getLongitude() {
		return longitude;
	}
	public void setLongitude(Double longitude) {
		this.longitude = longitude;
	}
    
    
}

File: android/app/src/main/java/com/ssafy/finalpass/dto/Store.kt
package com.ssafy.finalpass.dto

data class Store(
    val id: Int,
    val name: String,
    val latitude: Float,
    val longitude: Float,
    val comment: ArrayList<StoreComment> = ArrayList()
) {
    constructor(): this(0, "", 0f, 0f)
}
